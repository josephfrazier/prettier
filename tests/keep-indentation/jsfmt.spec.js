run_spec(__dirname, { keepIndentation: true, semi: false, printWidth: 80 });
run_spec(__dirname, { keepIndentation: true, semi: false, printWidth: 80, useTabs: true });
