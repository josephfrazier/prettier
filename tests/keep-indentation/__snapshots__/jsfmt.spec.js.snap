// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wrapping.js 1`] = `
					// this should not be wrapped
					1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567890
					// this should be wrapped when \`keepIndentation\` is enabled
					1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 12345678901
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          // this should not be wrapped
          1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567890
          // this should be wrapped when \`keepIndentation\` is enabled
          1234567 +
            1234567 +
            1234567 +
            1234567 +
            1234567 +
            1234567 +
            12345678901

`;

exports[`wrapping.js 2`] = `
					// this should not be wrapped
					1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567890
					// this should be wrapped when \`keepIndentation\` is enabled
					1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 12345678901
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
					// this should not be wrapped
					1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567 + 1234567890
					// this should be wrapped when \`keepIndentation\` is enabled
					1234567 +
						1234567 +
						1234567 +
						1234567 +
						1234567 +
						1234567 +
						12345678901

`;
